src/quest/manager.py

```diff
@@ -59,14 +62,19 @@ def our_handler(sig, frame):
         if self._storage.has_blob(self._namespace):
             self._workflow_data = self._storage.read_blob(self._namespace)
 
-        for wtype, wid, args, kwargs, background in self._workflow_data:
+        # Check storage to load stored workflow results from persistent storage
+        if self._storage.has_blob(f'{self._namespace}_results'):
+            self._results = self._storage.read_blob(f'{self._namespace}_results')
+
+        for wtype, wid, args, kwargs, background, start_time in self._workflow_data:
             self._start_workflow(wtype, wid, args, kwargs, background=background)
```

Comment:
We should make two functions:

```py
def _get_workflow_data_storage_key(self):
    return self._namespace

def _get_workflow_results_storage_key(self):
    return self._namespace + '_results'
```
And use these functions in`__aenter__` and `__aexit__` (or anywhere else it is necessary).